plugin-name/plugin-name.php

```diff
@@ -66,10 +59,5 @@ function deactivate_plugin_name() {
  *
  * @since    1.0.0
  */
-function run_plugin_name() {
-
-	$plugin = new Plugin_Name();
-	$plugin->run();
-
-}
-run_plugin_name();
+$plugin = new Plugin_Name();
```

Comment:
`$plugin` is scoped to the global namespace, just like `run_plugin_name` was.  But now the `Plugin_Name` instance is also scoped to the global namespace where before it was scoped to `run_plugin_name`.